src/main/java/pro/sky/CWork2/exception/BAD_REQUEST.java

```diff
@@ -0,0 +1,22 @@
+package pro.sky.CWork2.exception;
+
+public class BAD_REQUEST extends RuntimeException{
```

Comment:
```suggestion
public class BadRequestException extends RuntimeException{
```
классы называем в CamelCase и для исключений оканчиваем их на Exception